CHANGES.md

```diff
@@ -1,6 +1,473 @@
+* performance / benchmark / memory highlights::
+  - significant performance improvements when executing using Cray networks
+  - added a new faster version of the reverse-complement benchmark
+  - closed a number of minor memory leaks
+* portability highlights:
+  - ported Chapel to the Cray XC50 system with ARM processors
+  - improved portability to FreeBSD and PowerPC
+  - improved portability for specific versions of gcc
+* documentation highlights:
+  - moved Chapel's website from http://chapel.cray.com to https://chapel-lang.org
+  - added a color-coded version selection menu to the online documentation
+
+File / Directory Structure
+--------------------------
+* made `make install` install mason
+* added the CHPL_COMM_SUBSTRATE setting to GASNet-specific paths
+
+New Tools / Tool Changes
+------------------------
+* added support for bash autocompletion for `chpl` compiler options
+  (see https://chapel-lang.org/docs/1.17/usingchapel/compiling.html#tab-autocompletion-for-flags)
+* changed the generated executable's name from the main module to its filename
+  (see the `-o` flag on https://chapel-lang.org/docs/1.17/usingchapel/man.html)
+* added support for multiple mason repositories, including local registries
+  (see https://chapel-lang.org/docs/1.17/tools/mason/mason.html#local-registries)
+* added support for `mason help` and `mason version` commands
+* `chpl` now offers suggestions for unfamiliar flags
+  (see `chpl -fast`)
+* rewrote `printchplenv` and reworked its interface and output
+  (see `util/printchplenv --help`)
+* rewrote `compileline`
+  (see `util/config/compileline --help`))
+
+Semantic Changes / Changes to Chapel Language
+---------------------------------------------
+* `in` intents now behave more like variable initialization
+  (see https://github.com/chapel-lang/chapel/blob/master/doc/rst/developer/chips/13.rst)
+* ranges are now passed by `const in` intent by default
+  (see 'The Default Intent' in the 'Procedures' chapter of the specification)
+* disallowed implicit coercions for type arguments with a formal type specifier
+  (see 'Implicit Conversions' in the specification's 'Conversions' chapter)
+* changed the alignment of a non-stridable range from 0 to its low bound
+  (i.e., `(3..5).alignment` is now 3 where it used to be 0)
+
+New Features
+------------
+* added support for uninterpreted triple-quoted strings
+  (see 'Literals' section in 'Lexical Structure' chapter of the specification)
+* added support for `continue` statements within forall loops
+* extended `delete` to accept arrays and list of expressions to delete
+  (see 'Deleting Class Objects' in the 'Classes' chapter of the specification)
+* added an `isEmpty()` method on domains
+* type and param fields can now be accessed from a type variable
+  (see 'Field Accesses' in the 'Classes' chapter of the specification)
+* supported forwarding method calls on arrays, domains, and distributions
```

Comment:
`supported forwarding method calls` reads weirdly to me.
I could imagine it being `arrays, domains, and distributions now allow forwarded method calls` perhaps?